### DISCUSSION TITLE
Enable Cosmos to use TurboPack with Next.js

### DISCUSSION THREAD

**Discussion by weleo** [react-cosmos/react-cosmos#1532] [Ideas]
TurboPack speeds up Next.js development when enabled via the --turbo flag. Enabling Cosmos to use TurboPack with Next.js could streamline DX. Yay or nay?

**Comment by ovidiuch** [comment]
Haven't used Turbopack but that makes sense. Have you tried it with Next.js and Cosmos? I don't see why it wouldn't work.
